src/index.js

```diff
@@ -36,7 +37,29 @@ class App extends React.Component {
       behavior: "smooth"
     });
   };
-  componentDidMount() {}
+  handleScroll = () => {
+    const currentHeight = document.documentElement.scrollTop;
+    if (currentHeight > 3500) {
```

Comment:
3500 ??
will this value remain constant for different resume.json and different viewport size?